requirements.txt

```diff
@@ -16,3 +16,4 @@ pytest==2.9.1
 requests==2.9.1
 six==1.10.0
 Werkzeug==0.11.8
+beautifulsoup4
```

Comment:
That would be an easy fix on the first build. I feel adding `wikipedia==1.4.0` would be more explicit here because bs4 is going to be installed anyway. If any upcoming module uses bs4 and `wikipedia` is removed, we can simply add `beautifulsoup4==4.4.1`. Thoughts?